src/Controller/RunController.php

```diff
@@ -271,9 +271,11 @@ public function compare(Request $request): void
             'base_run' => $baseRun,
             'head_run' => $headRun,
             'candidates' => $candidates,
-            'url_params' => $request->get(),
+            // 'url_params' => $request->get(),
+            'url_params' => $request->getQueryParams(),
             'comparison' => $comparison,
             'paging' => $paging,
+            'pagination' => $paging,
```

Comment:
why is the `pagination` key added?

Reply:
compare.twig referencing a nonexistent value -> {{ helpers.pagination('run.compare', pagination, url_params) }} 
 i will remove old key `paging`.

Reply:
OR
{{ helpers.pagination('run.compare', pagination, url_params) }}
to
{{ helpers.pagination('run.compare', paging, url_params) }}

Reply:
other templates still use `paging`, so change to use `paging` in compare.twig as well:
- https://github.com/perftools/xhgui/blob/448c6d722440171612f9a257dcd6912fc940aae8/templates/runs/list.twig#L71

so, do the opposite, revert the changes in code, but fix template.

Reply:
unresolving, as not done as asked.

Reply:
done